runbotics/runbotics-desktop/src/core/bpm/runtime/runtime.service.ts

```diff
@@ -382,6 +382,7 @@ export class RuntimeService implements OnApplicationBootstrap, OnModuleDestroy {
             variables: {
                 ...request.variables,
                 tempFolder: this.getTempDirPath(),
+                userEmail: request.triggerData.userEmail,
```

Comment:
here you use request.triggerData.userEmail but in the interface userEmail is optional

whether add ? here or remove ? from the interface